test/runtests.jl

```diff
@@ -28,554 +33,746 @@ mycount(p, c) = p + 1
 bigsum(p, c) = p + big(c)
 sumpoint(p::Point3D, x, y, z) = p + Point3D(x, y, z)
 
-dbfile = joinpath(dirname(pathof(SQLite)), "../test/Chinook_Sqlite.sqlite")
-dbfile2 = joinpath(tempdir(), "test.sqlite")
-cp(dbfile, dbfile2; force=true)
-chmod(dbfile2, 0o777)
+test_dbfile = joinpath(tempdir(), "test.sqlite")
```

Comment:
shoudl we just move this line inside the `setup_clean_test_db` function?

Reply:
I use the variable in a couple of more basic tests that I wanted to keep transparent, I figured having a couple of simple tests makes catastrophic failures a bit easier to debug. :)